teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
+    @specialization = specialization
```

Comment:
- [ ] super(age, name, parent_permission), it could be simple as this you do not need to rename them as name: name,